content/components/action-bar.mdx

```diff
@@ -1,84 +1,89 @@
 ---
 
 import {Box, Text} from '@primer/react'
 
-## Overview
-
-Use an action bar to render multiple icon buttons in a row. Buttons can be split into groups by adding a divider. When there is not enough space, buttons that don't fit will be added to an overflow menu.
+<img
+  width="960"
+  alt=""
+  src="https://user-images.githubusercontent.com/378023/197716033-86fdac40-2b14-438c-a712-b003b76dabd3.png"
+/>
 
 ## Anatomy
 
+And action bar is mostly a wrapper for different kind of actions. It makes sure alliment and responsiveness works without much configurations.
+
 <img
   width="960"
   alt="A diagram of an action bar with a few buttons, a divider and at the end a button that opens an overflow menu."
-  src="https://user-images.githubusercontent.com/378023/193506131-be8bead4-d1ab-4c79-8ac1-054e75bac9be.png"
+  src="https://user-images.githubusercontent.com/378023/198211386-3e48eeb4-8cc6-41c0-85dd-1596aaf0d3ab.png"
 />
 
-## Content
+### Items
 
-### Buttons
+<Box
+  as="figure"
+  display={["block", "block", "block", "block", "grid"]}
+  gridTemplateColumns="1fr 1fr"
+  gridGap={7}
+  m={0}
+  mb={3}
+>
+  <Text as="p" mt="0">
+Each Action bar item is componsed of a visual and a label.
 
-An action bar should only contain icon buttons with the `invisible` variant (no border/background).
+- `visual`: An icon to best represent the item
+- `label`: Text describing the action of the item
 
-<DoDontContainer>
-  <Do>
-    <img
-      src="https://user-images.githubusercontent.com/378023/193506398-6d6da18a-b70d-4cd4-b6e4-fd472a24934f.png"
-      role="presentation"
-      width="456"
-    />
-    <Caption>Use only invisible icon buttons in an action bar.</Caption>
-  </Do>
-  <Dont>
+Note that depending on the variant used, the visual or label might not be visible.
+  </Text>
+  <Box>
     <img
-      src="https://user-images.githubusercontent.com/378023/193506488-44543352-f513-469e-8783-5d1cc7f44eaf.png"
+      src="https://user-images.githubusercontent.com/378023/198209541-7e647879-2096-46a3-b7bd-6d390ff72e41.png"
       role="presentation"
       width="456"
     />
-    <Caption>Don't use other variants or components in an action bar.</Caption>
-  </Dont>
-</DoDontContainer>
+  </Box>
+</Box>
 
 ### Dividers
 
-Dividers can be added to visually group related buttons.
-
-<DoDontContainer>
-  <Do>
-    <img
-      src="https://user-images.githubusercontent.com/378023/193506858-6542a6ba-4bac-400d-bba9-8601fbc032ed.png"
-      role="presentation"
-      width="456"
-    />
-    <Caption>Use a divider between buttons.</Caption>
-  </Do>
-  <Dont>
+<Box
+  as="figure"
+  display={["block", "block", "block", "block", "grid"]}
+  gridTemplateColumns="1fr 1fr"
+  gridGap={7}
+  m={0}
+  mb={3}
+>
+  <Text as="p" mt="0">
+    Use a divider to visually seperate items of the same type. No divider is needed between icons and buttons since they are already distinctive enough.
```

Comment:
```suggestion
    Use a divider to visually separate groups of items of the same type. No divider is needed between icons and buttons since they are already distinctive enough.
```

Reply:
Hmmm.. I think "groups of items of the same type" is not really accurate. A divider should only be used between items of the same type (e.g between icons or between buttons) but not between groups of the same type (e.g. between an icon group and a button group)

<img src="https://user-images.githubusercontent.com/378023/198211711-acca35ec-03d3-4ce1-9d02-05d65909e62f.png" role="presentation" width="456" />